node_openbsd.go

```diff
@@ -0,0 +1,36 @@
+package restic
+
+import (
+	"os"
+	"syscall"
+	"time"
+)
+
+func (node *Node) OpenForReading() (*os.File, error) {
+	file, err := os.OpenFile(node.path, os.O_RDONLY, 0)
+	if os.IsPermission(err) {
+		return os.OpenFile(node.path, os.O_RDONLY, 0)
+	}
+	return file, err
+}
+
+func (node *Node) createDevAt(path string) error {
+	return syscall.Mknod(path, syscall.S_IFBLK|0600, int(node.Device))
+}
+
+func (node *Node) createCharDevAt(path string) error {
```

Comment:
this and createDevAt are also identical to the generic ones in node.go. Why do we need BSD-specific copies that don't do anything BSD-specific?